lib/rust/ensogl/component/grid-view/src/column_widths.rs

```diff
@@ -76,10 +76,11 @@ impl ColumnWidths {
     /// `colunm` can be equal to `number_of_columns` passed to the [`Self::new`] or
     /// [`Self::resize`].
     pub fn pos_offset(&self, column: usize) -> f32 {
-        if column == 0 {
+        let borrowed = self.width_diffs.borrow();
+        if column == 0 || column > borrowed.len() {
```

Comment:
Is this a normal routine that someone reads position of non-existing column? If not, the error or warning should be printed.

Reply:
I did notice a panic in this place once. I'm not sure if this is a normal routine, so I left a warning here.